workflow-v2.1.0-e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0 e2e Tests (parallel)
+
+End-to-end tests for the Deis Workflow open source PaaS, executed in parallel.
+
+NOTE: This assumes a fresh `helmc install` of Deis v2. The test suite creates
+an initial user with admin privileges (who is deleted when tests complete).
+See https://github.com/deis/workflow-e2e for more detail.
+
+```console
+$ helmc uninstall -n deis -y workflow-e2e ; helmc install workflow-e2e
+$ kubectl --namespace=deis logs -f workflow-e2e tests
```

Comment:
> isn't the pod name configurable now?

Perhaps, but here's what my cluster looks like after `helmc generate && helmc install`:
```console
$ kd get po
NAME                          READY     STATUS      RESTARTS   AGE
deis-builder-3a8pt            1/1       Running     1          32m
deis-controller-xh5yw         1/1       Running     2          32m
deis-database-aps6j           1/1       Running     0          32m
deis-logger-fluentd-bl7il     1/1       Running     0          32m
deis-logger-fluentd-cqf5v     1/1       Running     0          32m
deis-logger-fluentd-ncgo1     1/1       Running     0          32m
deis-logger-qweh3             1/1       Running     2          32m
deis-minio-cmpde              1/1       Running     0          32m
deis-monitor-grafana-69gip    1/1       Running     0          32m
deis-monitor-influxdb-r71gs   1/1       Running     0          32m
deis-monitor-telegraf-0w4iz   1/1       Running     0          32m
deis-monitor-telegraf-epect   1/1       Running     0          32m
deis-monitor-telegraf-i35hg   1/1       Running     0          32m
deis-nsqd-tmflq               1/1       Running     0          32m
deis-registry-qscvs           1/1       Running     1          32m
deis-router-kjzk5             1/1       Running     0          32m
deis-workflow-manager-tzhjx   1/1       Running     0          32m
workflow-v2.1.0-e2e           1/2       Completed   0          22m
```
So either we change the pod name to `workflow-e2e` or update the command in the README to refer to `workflow-v2.1.0-e2e`.